### PR TITLE
basic UI

### PR DESCRIPTION
**Pull request by m05e5**
# OOP school library: add basic UI

Create a main.rb file that will serve as your console app entry-point. It should have methods that do the following:

    List all books.
    List all people.
    Create a person (teacher or student, not a plain Person).
    Create a book.
    Create a rental.
    List all rentals for a given person id.

In your main.rb define the entry point, this will be a method called main that is invoked at the end of your file. This method should do the following:

    Present the user with a list of options to perform.
    Lets users choose an option.
    If needed, ask for parameters for the option.
    Have a way to quit the app.

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+class App
+  def initialize()
+    @start = 'Please choose an option by entering a number
+    1 - List all books
+    2 - List all people
+    3 - Create a person
+    4 - Create a book
+    5 - Create a rental
+    6 - List all rentals for a given person id
+    7 - exit'
```
Comment (Davidosky007) [inline main.rb]: * I suggest you move all these options into a different method as in the project requirement your entry point should be in a method called `main`  [Refer back to the requirements](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_ui.md#:~:text=In%20your%20main,quit%20the%20app.)

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
## Hi @m05e5 👋, 

Good job so far! 💪 
* Descriptive commit messages ✔️ 
* No linter errors ✔️ 
* Well written PR ✔️ 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.rb [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+  end
+  # rubocop:enable Metrics/CyclomaticComplexity
+end
+
+def main
+  app = App.new
+  app.run
+end
+
+main
```
Comment (m05e5) [inline main.rb:134]: Please as in the requirement and the documentation microverse gave us this is my entry point which is the main method

Comment (m05e5) [reply main.rb:134]: And each of the options provided is in a separate method of the App class.

Comment (m05e5) [reply main.rb:134]: So if I'm wrong please explain me better my mistake

File: main.rb (hunk 1 above)
Comment (Davidosky007) [reply main.rb]: * The `main method` should house the following options, please check the screenshot below ⬇️.  when i start the app the main method should be your entry point 
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
## Hi @m05e5 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Davidosky007** [comment]
## Hi @m05e5 👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 
job-welldone-gif




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**